### PR TITLE
Rake.last_comment was depreciated in Rake11.

### PR DESCRIPTION
**Pull request by gkuchta**
See https://github.com/ruby/rake/commit/e76242ce7ef94568399a50b69bda4b723dab7c75

### CODE REVIEW

**Comment by rcyrus** [comment]
:+1:

File: lib/foodcritic/version.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module FoodCritic
   # The current version of foodcritic
-  VERSION = '6.0.1'
+  VERSION = '6.0.2'
```
Comment (tas50) [inline lib/foodcritic/version.rb]: Can you remove this.  We'll handle the version bump when we're ready for a release

Comment (gkuchta) [inline lib/foodcritic/version.rb]: @tas50 sure thing. i've removed the version bump.

**Comment by tas50** [comment]
@gkuchta Can you rebase this down to a single commit too so the version doesn't bump in git history?

Thanks,
Tim

**Comment by gkuchta** [comment]
@tas50 sure thing. i rebase'd off 7d512af31aebbf978f1be680cc519b84cef6af30 -- looking cleaner now?

**Pull request merged**